### DISCUSSION TITLE
Leave App open or not (?)

### DISCUSSION THREAD

**Discussion by felixvklein** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1168] [General]
Hi guys, 

I have the free version of AlDente and I was wondering if I need to have the app open all the time in order to work.. Couldn't find an answer to that.

Hoping that makes sense.

**Comment by MatthiasKerbl** [comment]
Hi @felixvklein,

Yes, the free version needs to be running at all times for the charge limit to stay active.

Best,
Matthias
